changelog/unreleased/tokenInfo-endpoint-replacement.md

```diff
@@ -0,0 +1,12 @@
+Enhancement: Replacement for TokenInfo Endpoint
+
+      
+The client should basically always send a PROPFIND to /dav/public-files/{sharetoken}
+
+* authenticated clients accessing an internal link are redirected to the "real" resource (`/dav/spaces/{target-resource-id}
+* authenticated clients accessing a public link (password protected or not) for a resource they already have access to are also redirected to the "real" resource. (and always need to supply the password)
```

Comment:
I think this is wrong. We're currently not creating redirect responses when an authenticate client access a public link. But that is ok. So please just remove the item from this list.

Reply:
How about this wording?
`authenticated clients accessing a public link (password-protected or not) able to resolve public shares like before (need to supply the password for password-protected public link).`

Reply:
Hm, I can't quite parse that sentence. :thinking: I think there is is verb missing somewhere.

How about:
"authenticated clients are able to resolve public links like before. For password protected links they need to supply the password even if they have access to the underlying resource by other means."